solr/webapp/web/WEB-INF/web.xml

```diff
@@ -140,6 +144,11 @@
   </servlet-mapping>
   <!-- Fullstory additions -->
 
+  <servlet-mapping>
+    <servlet-name>PrometheusStatz</servlet-name>
+    <url-pattern>/admin/prometheusMetrics</url-pattern>
+  </servlet-mapping>
+
```

Comment:
put this up in Fullstory additions comment?
Also, should this be mounted at `/metrics` maybe for consistency?

Reply:
just moved it up, and it can't be mounted at metrics because there's already an xml page at metrics for solr that is presumably built-in from somewhere else?

Reply:
Weird, because https://solr-c01-1.internal.fullstory.com/solr/metrics is a 404 for me

Reply:
It's at solr/admin/metrics - I can change it to just solr/metrics if that is more appropriate

Reply:
Perfect!